kivymd/tools/release/make_release.py

```diff
@@ -157,14 +157,22 @@ def move_changelog(
 
 
 def create_unreleased_changelog(
-    index_file, unreleased_file, version, ask: bool = True, test: bool = False,
+    index_file,
+    unreleased_file,
+    version,
+    ask: bool = True,
+    test: bool = False,
 ):
     """Create unreleased.rst by template."""
     # Check if unreleased file exists
     if os.path.exists(unreleased_file):
         if ask and input(
             f'Do you want to rewrite "{unreleased_file}"? (y)'
-        ) not in ("", "y", "yes",):
+        ) not in (
+            "",
+            "y",
+            "yes",
+        ):
             exit(0)
     # Generate unreleased changelog
     changelog = f"""Unreleased
```

Comment:
Updated by pre-commit of github actions. 8994a39cd2b1050aa6b8342b533d148cbe2e065a